### ISSUE TITLE
Improvement - CORS error in local development

### ISSUE THREAD

**Issue by pseudovirtual** [draftedus/blm-generator#5]
When testing the script generated on my local, I get a CORS error in the console, which only appears to happen for local development but not for production environments. This is likely because the origin is null when running the script locally. Ideally, the script would just say that it's working and print a helpful message telling me it's ok on my local and that it will work in production.

**Comment by samhagman** [comment]
Yeah good idea, I can see how this repeated error in the console could be annoying and/or concerning during local development. We can look into making the local testing setup better. 👍
